lib/matplotlib/animation.py

```diff
@@ -1299,7 +1299,20 @@ def to_html5_video(self, embed_limit=None):
             return 'Video too large to embed.'
 
     def to_jshtml(self, fps=None, embed_frames=True, default_mode=None):
-        """Generate HTML representation of the animation"""
+        """
+        Generate HTML representation of the animation.
+
+        Parameters
+        ----------
+        fps : int, optional
+            Movie frame rate (per second). If not set, the frame rate from
+            the animation's frame interval.
+        embed_frames : bool, optional
```

Comment:
From a quick look: If `embed_frames` is True, the frame image data are inlined into the HTML. Otherwise they are written to separate files. Given that the data is read back in, I'm wondering if that option does make any sense (or works if set to false at all), becuase that would imply we still need access to the temporary directory. - This should probably be tested.

Reply:
I don't have bandwidth to check at the moment - if someone else wants to feel free, otherwise I'd advocate merging this as an improvement over what was there before.